firefox_desktop/explores/crashes.explore.lkml

```diff
@@ -26,14 +18,240 @@ explore: crash_usage {
+        content_crash_rate,
+        content_crashes,
+        content_crash_incidence,
+        gpu_crash_rate,
+        gpu_crashes,
+        gpu_crash_incidence,
+        main_crash_rate,
+        main_crashes,
+        main_crash_incidence,
+        rdd_crash_rate,
+        rdd_crashes,
+        rdd_crash_incidence,
+        socket_crash_rate,
+        socket_crashes,
+        utility_crash_incidence,
+        utility_crash_rate,
+        utility_crashes,
+        vr_crash_rate,
+        vr_crashes,
+        vr_crash_incidence,
+      ]
+      filters: [
+        crash_usage.days_since_build_published: "<=56",
+        crash_usage.ten_percent_sample: "No",
+        crash_usage.channel: "nightly"
+      ]
+    }
+    materialization: {
+      sql_trigger_value: SELECT MAX(submission_date) FROM `moz-fx-data-shared-prod`.telemetry_derived.clients_daily_joined_v1 ;;
+      increment_key: "submission_date"
+    }
+  }
+
+  aggregate_table: full_nightly_buildid_rollups {
+    query: {
+      dimensions: [
+        build_id,
+        build_id_as_int,
+        channel,
+        os_name,
+      ]
+      measures: [
+        active_hours,
+        content_crash_rate,
+        gpu_crash_rate,
+        main_crash_rate,
+        rdd_crash_rate,
+        socket_crash_rate,
+        utility_crash_rate,
+        vr_crash_rate,
+      ]
+      filters: [
+        crash_usage.submission_date: "3 month",
+        crash_usage.days_since_build_published: "<=56",
+        crash_usage.ten_percent_sample: "No",
+        crash_usage.channel: "nightly"
+      ]
+    }
+
+    materialization: {
+      sql_trigger_value: SELECT MAX(submission_date) FROM `moz-fx-data-shared-prod`.telemetry_derived.clients_daily_joined_v1 ;;
+    }
+  }
+
+
+  aggregate_table: full_beta_daily_rollups {
+    query: {
+      dimensions: [
+        submission_date,
+        version,
+        major_version,
+        minor_version,
+        os_name,
+      ]
+      measures: [
+        active_hours,
+        dau_count,
+        content_crash_rate,
+        content_crashes,
+        content_crash_incidence,
+        gpu_crash_rate,
+        gpu_crashes,
+        gpu_crash_incidence,
+        main_crash_rate,
+        main_crashes,
+        main_crash_incidence,
+        rdd_crash_rate,
+        rdd_crashes,
+        rdd_crash_incidence,
+        socket_crash_rate,
+        socket_crashes,
+        utility_crash_incidence,
+        utility_crash_rate,
+        utility_crashes,
+        vr_crash_rate,
+        vr_crashes,
+        vr_crash_incidence,
+      ]
+      filters: [
+        crash_usage.days_since_build_published: "<=56",
+        crash_usage.ten_percent_sample: "No",
+        crash_usage.channel: "beta"
+      ]
+    }
+    materialization: {
+      sql_trigger_value: SELECT MAX(submission_date) FROM `moz-fx-data-shared-prod`.telemetry_derived.clients_daily_joined_v1 ;;
+      increment_key: "submission_date"
+    }
+  }
+
+  aggregate_table: full_beta_buildid_rollups {
```

Comment:
This seems almost identical to `full_nightly_buildid_rollups` besides the filter, could that filter just be added as a dimension? Could turn two queries into one.

Reply:
It is almost identical. What I'm trying to avoid is a daily rollup of 100% of release, and there's no clear way to do that with a single aggregate table that also lets you specify either nightly _or_ beta. (The problem is that Looker requires filters to exactly match. It's been a real headache trying to make sure these aggregates get used by the tiles in the stability dashboard).